externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/ExecutePython.scala

```diff
@@ -74,6 +74,16 @@ class ExecutePython(
         s"($statementId, $redactedStatement, $forceCancel)")
       setSparkLocalProperties(KYUUBI_SESSION_USER_KEY, session.user)
       setSparkLocalProperties(KYUUBI_STATEMENT_ID_KEY, statementId)
+
+      if (isSessionUserVerifyEnabled) {
+        setSparkLocalProperties(
+          KYUUBI_SESSION_SIGN_PUBLICKEY,
+          session.conf.get(KYUUBI_SESSION_SIGN_PUBLICKEY).orNull)
+        setSparkLocalProperties(
+          KYUUBI_SESSION_USER_SIGN,
+          session.conf.get(KYUUBI_SESSION_USER_SIGN).orNull)
```

Comment:
If `isSessionUserVerifyEnabled`, `KYUUBI_SESSION_SIGN_PUBLICKEY` `KYUUBI_SESSION_USER_SIGN` must be present?

Reply:
Yes, these two configs are set to session via `configOverlay`  in `OpenSession` on server side , when session sign enabled.

Reply:
my point is should we fail fast instead of `.orNull`?